### ISSUE TITLE
Use Case - Support other Digital Formats for Books

### ISSUE THREAD

**Issue by Natkeeran** [Islandora-CLAW/CLAW#798]
Islandora 7.x tightly couples the [Content Model](https://github.com/Islandora/islandora/wiki/Content-Models) with [Digital/Content Format](https://www.loc.gov/preservation/digital/formats/content/content_categories.shtml).  A book can be in many digital formats ([example](https://archive.org/details/theprinciplesofp01jameuoft)), including EPUB, PDF, plain text and other formats.  Current release tightly couples the book with the paged content view and the archival viewer.  For example, out of the box, it does not support a pdf book format (without generating the large image derivatives).  Large Image derivatives can be problematic, because they are resource intensive to create, store and serve.  

In the Book/Newspaper case, the content model should not be tied to the viewer/paged content.  The content model should be abstract and support other digital formats such as pdf or text.

It would be good to describe what are the best practices in building/relating Content Models with Digital Formats with Metadata Profiles with Viewers/Displays.  

Related use case: https://github.com/Islandora-CLAW/CLAW/issues/688

**Comment by dannylamb** [comment]
Yes... wow... now I see those things have been conflated and we have an opportunity to detangle them.  Fantastic use case @Natkeeran 👍 👍 👍

**Comment by ajs6f** [comment]
:+1:  x 1000. Don't make my metadata depend on file formats!

**Comment by mjordan** [comment]
Totally. Another example of this binding in 7.x is that a PDF is not as much a "content model" in 7.x as it is a "MIME type model". A lot of different types of content can be represented as a PDF file - in image, a document, a book, a newspaper issue, etc.

**Comment by Natkeeran** [comment]
@mjordan 

Yes, MIME type model is a good way to put it.  Islandora Scholar (Citation, Thesis Content Models) uses pdfs, even though it does not use PDF content model.  

End users currently create a collection containing multiple content models or a custom module to include multiple type of MIME types in a single collection.  The difficulty is pdf metadata profile does not always work for a book.

**Comment by kstapelfeldt** [comment]
#688
